.github/workflows/pull-request.yml

```diff
@@ -87,22 +111,28 @@ jobs:
               with:
                   node-version: ${{ env.NODE_VERSION }}
 
-            - name: Cache dependencies
-              id: npm_cache
+            - name: Use pnpm
+              uses: pnpm/action-setup@v2.2.4
+              with:
+                  version: latest
+                  run_install: false
+
+            - name: Get pnpm store directory
+              id: pnpm-cache
+              run: echo "::set-output name=pnpm_cache_dir::$(pnpm store path)"
+
+            - name: Setup pnpm cache
               uses: actions/cache@v3
               with:
                   path: |
-                      node_modules
-                      ${{ env.CYPRESS_CACHE_FOLDER }}
-                  key: node_modules-${{ hashFiles('package-lock.json') }}
-                  restore-keys: node_modules-
+                      ${{ steps.pnpm-cache.outputs.pnpm_cache_dir }}
+                      ~/.cache/Cypress
+                  key: pnpm-with-cypress-${{ runner.os }}-${{ hashFiles('pnpm-lock.yaml') }}
+                  restore-keys: pnpm-with-cypress-${{ runner.os }}-${{ hashFiles('pnpm-lock.yaml') }}
 
             - name: Install dependencies
               if: steps.npm_cache.outputs.cache-hit != 'true'
-              run: npm ci
+              run: pnpm install --frozen-lockfile
```

Comment:
So `pnpm install --frozen-lockfile` is the new `npm ci`, should we run it everytime we update/switch our branches? Or pnpm install is enough? Otherwise it would be nice to have a script with a shorter name for it. 😅 no?

Reply:
> should we run it everytime we update/switch our branches? Or pnpm install is enough?

`pnpm i` is enough